backend/app/routers/exhibitors.py

```
@@ -0,0 +1,112 @@
+    await db.commit()
+    await db.refresh(e)
+    contact = await _load_contact(db, e.contact_person_id)
+    return exhibitor_to_dict(e, contact)
+
+
+@router.delete(
+    "/{exhibitor_id}",
+    status_code=status.HTTP_204_NO_CONTENT,
+    dependencies=[Depends(require_admin)],
+)
+async def delete_exhibitor(exhibitor_id: int, db: AsyncSession = Depends(get_db)) -> None:
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)